__tests__/components/Card.test.tsx

```diff
@@ -9,9 +9,12 @@
 // WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 // PARTICULAR PURPOSE. See the GNU General Public License for more details.
 
-import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import { cleanup, render, screen } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
```

Comment:
The `userEvent` package is useful for interacting with tooltips, number inputs and similar composite components. The `fireEvent` api from react testing library doesn't trigger `mouseover` / `change` events for those components correctly.